backend/src/config/index.ts

```diff
@@ -69,11 +69,13 @@ const email = {
   EMAIL: process.env.EMAIL === 'true' || false,
   EMAIL_TEST_MODUS: process.env.EMAIL_TEST_MODUS === 'true' || false,
   EMAIL_TEST_RECEIVER: process.env.EMAIL_TEST_RECEIVER || 'stage1@gradido.net',
-  EMAIL_USERNAME: process.env.EMAIL_USERNAME || 'gradido_email',
+  EMAIL_USERNAME: process.env.EMAIL_USERNAME || 'null',
```

Comment:
please no string

Reply:
This is fixed in https://github.com/gradido/gradido/commit/1e265b1078c10cfb2a7aeaebeaecbca4e6983780.